environment-dev.yml

```diff
@@ -39,6 +39,9 @@ dependencies:
   - mypy<0.990
   - pandas-stubs
   - fastparquet
+  # for release script
+  - pygithub
```

Comment:
We already have pygithub at 31 line

Reply:
good catch, I've removed extra line and upinned `pygithub` version everywhere (it's ancient and not actively used in our codebase except the release script, so should be fine to go with a recent version)